standalone-metastore/metastore-common/src/main/thrift/hive_metastore.thrift

```diff
@@ -2679,7 +2742,8 @@ PartitionsResponse get_partitions_req(1:PartitionsRequest req)
 
   list<string> get_partition_names(1:string db_name, 2:string tbl_name, 3:i16 max_parts=-1)
```

Comment:
shouldnt this be deleted?

Reply:
We'll deprecate the older APIs in the next release.